### PR TITLE
Collect metrics for FAQ browsing

### PR DESCRIPTION
**Pull request by gsvr**
- [x] Every time an FAQ topic is viewed increment `faq_view_topic_topicname` 
- [x] Every time an FAQ question is viewed increment `faq_view_question` 
- [x] Every time an FAQ question is sent via SMS increment `faq_sent_via_sms`

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by imsickofmaps** [comment]
@gsvr great, apart from can we add a negative test for metric fires too to ensure we're not double counting, just like we want to avoid double SMS sending?

**Comment by gsvr** [comment]
@imsickofmaps Added a few features - re-review please.

File: src/browser.js [hunk 1]
```diff
@@ -82,11 +83,21 @@ go.app = function() {
...
-                            name: 'states_questions',
-                            creator_opts: {
-                                faq_id: opts.faq_id
-                            }
+                        next: function(choice) {
+                            return self.im.metrics.fire
+                                .inc([
+                                        self.env,
+                                        'faq_view_topic',
+                                        choice.label.toLowerCase()
```
Comment (imsickofmaps) [inline src/browser.js]: This could have spaces in it too which will break metric store

**Comment by imsickofmaps** [comment]
@gsvr looks good, I think go back to ID's though on the metric name for topics. Thinking further, there's so many things that you'd need to protect against (unicode etc.) that it's a rabbit hole.

**Comment by gsvr** [comment]
@imsickofmaps :+1: ?

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**